e2e/test/scenarios/embedding-sdk/editable-dashboard.cy.spec.js

```diff
@@ -0,0 +1,65 @@
+import {
+  restore,
+  setTokenFeatures,
+  visitFullAppEmbeddingUrl,
+} from "e2e/support/helpers";
+import {
+  EMBEDDING_SDK_STORY_HOST,
+  describeSDK,
+} from "e2e/support/helpers/e2e-embedding-sdk-helpers";
+import {
+  JWT_SHARED_SECRET,
+  setupJwt,
+} from "e2e/support/helpers/e2e-jwt-helpers";
+
+describeSDK("scenarios > embedding-sdk > editable-dashboard", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    setTokenFeatures("all");
+    setupJwt();
+    cy.request("PUT", "/api/setting", {
+      "enable-embedding-sdk": true,
+    });
+
+    cy.createDashboard(
+      {
+        name: "Embedding SDK Test Dashboard",
+      },
+      { wrapId: true },
+    );
+
+    cy.signOut();
+
+    cy.intercept("GET", "/api/dashboard/*").as("getDashboard");
+    cy.intercept("GET", "/api/user/current").as("getUser");
+    cy.intercept("POST", "/api/dashboard/*/dashcard/*/card/*/query").as(
+      "dashcardQuery",
+    );
+  });
+
+  it("Should not open sidesheet when clicking last edit info (metabase#48354)", () => {
+    cy.get("@dashboardId").then(dashboardId => {
+      visitFullAppEmbeddingUrl({
+        url: EMBEDDING_SDK_STORY_HOST,
+        qs: {
+          id: "embeddingsdk-editabledashboard--default",
+          viewMode: "story",
+        },
+        onBeforeLoad: window => {
+          window.JWT_SHARED_SECRET = JWT_SHARED_SECRET;
+          window.METABASE_INSTANCE_URL = Cypress.config().baseUrl;
```

Comment:
out of this PR's scope, but I wonder if we should probably have a shorthand for this

Reply:
I'm not sure what you're imagining, can you give an example?

Reply:
@WiNloSt I'm thinking about a shared function that we can use that configures SDK test, i.e. setting the `window.*` variables and configures embedding, in tandem with [the below comment](https://github.com/metabase/metabase/pull/48630#discussion_r1800711857). I think we're moving towards component testing as you mentioned, so don't need to do it now.

Reply:
Yeah, this setup is the first iteration we came up with for SDK testing. I don't think it's solid enough to be a stepping foundation though.